### PR TITLE
Holy water song

### PR DESCRIPTION
**Pull request by igachungi**
Adding the lyrics holy water

### CODE REVIEW

File: lyrics/pop/holy water.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Wash away your sins, I sip away my sins
+And she said
+"Oh, my daughter, oh, I love you"
+Sip away your sins
+But I do what I want so I guess I'm gonna die young
+
+When I get home will you leave the lights on?
+'Cause it gets a little dark in here, dark in here
+I've been trying to get high fix the pieces in my mind
+'Cause I'm on a low down here
```
Comment (igachungi) [inline lyrics/pop/holy water.txt:11]: this looks good

File: lyrics/pop/holy water.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Oh, my mother, holy water
+Wash away your sins, I sip away my sins
+And she said
+"Oh, my daughter, oh, I love you"
+Sip away your sins
+But I do what I want so I guess I'm gonna die young
+
+When I get home will you leave the lights on?
```
Comment (rhyslim) [inline lyrics/pop/holy water.txt:8]: How about changing "lights on" to leave the door open? @igachungi

File: lyrics/pop/holy water.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Oh, my mother, holy water
+Wash away your sins, I sip away my sins
+And she said
+"Oh, my daughter, oh, I love you"
+Sip away your sins
+But I do what I want so I guess I'm gonna die young
```
Comment (rhyslim) [inline lyrics/pop/holy water.txt:6]: Let's say die old, not die young

Comment (igachungi) [reply lyrics/pop/holy water.txt:6]: I agree

File: lyrics/pop/holy water.txt (hunk 2 above)
Comment (igachungi) [reply lyrics/pop/holy water.txt:8]: Sure!

**Pull request merged**